### DISCUSSION TITLE
Why Not Using JS Frameworks for the Front Side?

### DISCUSSION THREAD

**Discussion by Strahfe** [collove/pasteme#72] [Q&A] [answered]
You've already implemented a RESTful service that contains all parts of PasteMe. Why don't you head to JS frameworks such as ReactJs and VueJs for a better design?

Thank you for making this awesome tool btw. That's so cool.

**Comment by farhaduneci** [comment, marked as answer]
Hello Arshia,
I hope you're doing well,

If I want to give you a quick answer on this one, we're back-end developers with simple/basic knowledge on the front-end aspect of this project.

> We for sure like to have a JS framework up and running as our front-end.

Meanwhile, everything is ready and a lot simpler to use for us in Django-templates, so that's why we're using it. Django template engine provides everything we've needed so far, and moving to JS frameworks will be a lot of work for the team.

Hope this will help. Thanks :white_heart:

**Reply by Strahfe** [reply]
Thanks. Hope you find your awesome front-end devs.
